jupyterhub/handlers/login.py

```diff
@@ -89,8 +89,8 @@ async def get(self):
 class LoginHandler(BaseHandler):
     """Render the login page."""
 
-    def _render(self, login_error=None, username=None):
-        return self.render_template(
+    async def _render(self, login_error=None, username=None):
```

Comment:
FWIW, you don't need to add this `async def` and `await` to a method that only wraps a function already returning an awaitable. The existing method will work fine and pass through the awaitable without needing two stacked awaits, one for the wrapper and another for the caller

Reply:
Done!